package.json

```diff
@@ -53,5 +53,11 @@
   },
   "standard": {
     "parser": "babel-eslint"
-  }
+  },
+  "files": [
+     "LICENSE",
+     "README.md",
+     "src/",
```

Comment:
There is no need to put the source files on npm (although this was done previously).